ethcore/light/src/client/header_chain.rs

```diff
@@ -74,6 +74,38 @@ pub struct BlockDescriptor {
 	pub total_difficulty: U256,
 }
 
+// best block data
+struct BestAndLatest {
```

Comment:
handwritten implementations of `Encodable` and `Decodable` can be replaced with 

```
#[derive(RlpEncodable, RlpDecodable)]
struct {
}
```

Reply:
Of course! Fixed.